app/Http/Requests/UpdateCourseRegistrationRequest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Enums\RegistrationStatus;
+use App\ValueObjects\UpdateCourseRegistration;
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateCourseRegistrationRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'status' => [
+                'required',
+                'string',
+                'in:'.implode(',', [RegistrationStatus::ACCEPTED->value, RegistrationStatus::REJECTED->value]),
```

Comment:
Tutaj można poza hard codowaniem wartości, stworzyć metodę w Enumie zwracająca te opcje które są dostępne a następnie coś w stylu:
Rule::in(array_column(RegistrationStatus::nazwa_metody(), 'value'))],